README.md

```diff
@@ -506,7 +529,7 @@ Options:
   --vs-api-key <key>                          API key, to authenticate against the ValueSet service to be used for resolving missing ValueSets.
   --focused-statement <statement expression>  Top level statement expression, i.e. "Initial Population", used to narrow the focus of a queryInfo calculation to just that statement and any children
   --cache-valuesets                           Whether or not to cache ValueSets retrieved from the ValueSet service. (default: false)
-  --trust-meta-profile                        To "trust" the content of meta.profile as a source of truth for what profiles the data that cql-exec-fhir grabs validates against. (default: true)
+  --trust-meta-profile <true|false>           Indicates whether to trust the meta.profile field in input FHIR resources as the authoritative source for profile validation. (default: true)
```

Comment:
I like this wording. I think just for consistency, can we also update the description for `trustMetaProfile` in the #calculation-options section of the README and in the calculator option comment description in `src/types/Calculator.ts` so that it matches this?

Reply:
@zacharyrobin - just bumping this to also update the calculator option comment description in `src/types/Calculator.ts`!